Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml

```diff
@@ -2507,18 +2522,415 @@ script:
   - arguments: []
     description: Updates the remote incident or detection with local incident or detection changes. This method is only used for debugging purposes and will not update the current incident or detection.
     name: update-remote-system
-  dockerimage: demisto/python3:3.10.8.37753
+  - arguments:
+    - description: limit the vulnerabilities returned to specific properties. Each value must be enclosed in single quotes and placed immediately after the colon with no space.
+      name: filter
+      required: false
+    - description: Unique agent identifier (AID) of a sensor
```

Comment:
Where can I get the AID?

Reply:
it would be in the context-data results after running the command 'cs-falcon-spotlight-search-vulnerability'

Reply:
ok Let's add this to the description. 
Also maybe its worth adding it to the human-readable as well since its been used in another commands